test/use-ref-value.test.js

```diff
@@ -74,7 +74,7 @@ describe('trim-name', () => {
       original: {
         value: ['{shadow.core.4}', 'pre-transformed-shadow-2'],
       },
-      value: ['{shadow.core.4}', 'pre-transformed-shadow-2'],
+      value: ['{shadow.core.4}', '0 0 2px rgba(0,0,0,0.6)'],
```

Comment:
I believe this was expected object was faulty before, as it also reverted the transform done on the original value.